test_conformance/basic/test_fpmath.cpp

```diff
@@ -106,7 +106,7 @@ int verify_fp(std::vector<T> (&input)[2], std::vector<T> &output,
         if (r != output[i] && nan_test)
         {
             log_error("FP math test for type: %s, vec size: %zu, failed at "
-                      "index %d, %a '%c' %a, expected %a, get %a\n",
+                      "index %u, %a '%c' %a, expected %a, get %a\n",
```

Comment:
suggestion that we could use `%zu` rather than `%u` for `size_t`

Reply:
Good catch, thanks!  We really need to get `Wformat` fixed and enabled. :-)